### ISSUE TITLE
Please provide options to exclude some values from the generated git.properties

### ISSUE THREAD

**Issue by tha2015** [n0mer/gradle-git-properties#3]
I'm like to exclude some properties from generated git.properties 

git.commit.user.email
git.commit.user.name
git.commit.message.short
git.commit.message.full

Please provide options to exclude these properties. Thanks.

**Comment by n0mer** [comment]
@tha2015 what properties do you actually want to see and care about?

**Comment by tha2015** [comment]
Currently the plugin always generates 8 below properties

git.commit.id
git.commit.time
git.commit.id.abbrev
git.branch
git.commit.user.name
git.commit.user.email
git.commit.message.short
git.commit.message.full

But I just want to 4 properties as below
git.commit.id
git.commit.time
git.commit.id.abbrev
git.branch

I don't want to 4 other properties
git.commit.user.name
git.commit.user.email
git.commit.message.short
git.commit.message.full

The reason is because I don't want to make visible some information in the output JAR file.

Thanks for the plugin.

**Comment by n0mer** [comment]
@tha2015  please try `1.4.9` version, it now has new config parameter `keys`.

if omitted, then by default all keys will be printed.

if you define 
```
gitProperties {
    keys = ['git.branch', 'git.commit.id', 'git.commit.id.abbrev', 'git.commit.time']
}
```

then plugin will put only these properties.

**Comment by tha2015** [comment]
I got the below error

* What went wrong:
A problem occurred evaluating root project 'ops-root'.
> Cannot cast object '[git.branch, git.commit.id, git.commit.id.abbrev, git.commit.time]' with class 'java.util.ArrayList' to class 'java.util.Map' due to: groovy.lang.GroovyRuntimeException: Could not find matching constructor for: java.util.Map(java.lang.String, java.lang.String, java.lang.String,
 java.lang.String)

I think it is because keys is defined as a Map in the below class
```groovy
 class GitPropertiesPluginExtension {
    ...
    Map keys
}
```
I think it should be defined as  List instead

**Comment by n0mer** [comment]
@tha2015 , you're right - please try just published v`1.4.10`

**Comment by tha2015** [comment]
It works well now. Thank you very much.
